modules/monitoring-setting-audit-log-levels-for-the-prometheus-adapter.adoc

```diff
@@ -0,0 +1,100 @@
+// Module included in the following assemblies:
+//
+// * monitoring/configuring-the-monitoring-stack.adoc
+
+:_content-type: PROCEDURE
+[id="setting-audit-log-levels-for-the-prometheus-adapter_{context}"]
+= Setting audit log levels for the Prometheus Adapter
+
+[role=_abstract]
+In default platform monitoring, you can configure the audit log level for the Prometheus Adapter.
+
+.Prerequisites
+
+* You have installed the OpenShift CLI (`oc`).
+* You have access to the cluster as a user with the `cluster-admin` role.
+* You have created the `cluster-monitoring-config` `ConfigMap` object.
+
+.Procedure
+
+You can set an audit log level for the Prometheus Adapter in the default `openshift-monitoring` project:
+
+. Edit the `cluster-monitoring-config` `ConfigMap` object in the `openshift-monitoring` project:
++
+[source,terminal]
+----
+$ oc -n openshift-monitoring edit configmap cluster-monitoring-config
+----
+
+. Add `profile:` under `audit` in the `k8sPrometheusAdapter` section under `data/config.yaml`:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cluster-monitoring-config
+  namespace: openshift-monitoring
+data:
+  config.yaml: |
+    k8sPrometheusAdapter:
+      audit:
+        profile: <audit_log_level> <1>
+----
+<1> The audit log level to apply to the Prometheus Adapter. 
+
+. Set the audit log level by using one of the following values for the `profile:` parameter:
++
+* `None`: Do not log events.
+* `Metadata`: Log only the metadata for the request, such as user, timestamp, and so forth. The request text and the response text will not be logged.
```

Comment:
`Metadata` is the default audit level, we'd better mention it

Reply:
Added a sentence to the description indicating that `Metadata` is the default value.